oktapam/data_source_cloud_connection_test.go

```diff
@@ -0,0 +1,63 @@
+package oktapam
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+)
+
+func TestAccDataSourceCloudConnection(t *testing.T) {
+	checkTeamApplicable(t, true)
+	identifier := randSeq()
+	initConfig := createTestAccDataSourceCloudConnectionInitConfig(identifier)
+	fetchConfig := testAccDataSourceCloudConnectionConfig("data1", identifier+"-1", "data2")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviders,
+		CheckDestroy:      testAccCloudConnectionsCheckDestroy(identifier+"-1", identifier+"-2"),
```

Comment:
i think you only need to check for `identifier+"-1"` since you're not creating a second resource.

Reply:
Done.